syntax = "proto3";

option java_multiple_files = true;
option java_package = "com.ericsson.adp.mgmt.metadata";

message Fragment {
    string fragmentId = 1;
    string version = 2;
    string sizeInBytes = 3;
    map<string, string> customInformation = 4;
}